### ISSUE TITLE
3.3 - (Exploit) Recon Unit Barb Pull

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#9769]
_1. Mod Version (X.Y.Z). Current Version: 3.3.1_

3.3

_2. DirectX Version_

_3. List of Other Mods_

[ModList2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/11236726/ModList2.zip)

_4. Describe the Issue_

Similar to the report I gave before: https://github.com/LoneGazebo/Community-Patch-DLL/issues/9726

This time I am able to draw a barb away from their camp and just take it with a warrior, easy peesy.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

**Saved Game**
[(Exploit) Luring Barbs from camp.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/11236731/Exploit.Luring.Barbs.from.camp.zip)

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

**Logs**
[Logs - Copy.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/11236736/Logs.-.Copy.zip)

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by ilteroi** [comment]
crashes on load, i hate this game

**Comment by Stalker012345** [comment]
> crashes on load, i hate this game
> 
> edit: infoaddict again; i thought that problem was fixed.
> 
> anyhow, the problem is not that they leave the camp, it's that they never move back in, right?

Correct, they won't defend their camp even when I literally will snag it on the next move.
